src/plugins/region_map/public/plugin.ts

```diff
@@ -58,11 +58,14 @@ import { OpenSearchDashboardsLegacyStart } from '../../opensearch_dashboards_leg
 import { SharePluginStart } from '../../share/public';
 
 /** @private */
-interface RegionMapVisualizationDependencies {
+export interface RegionMapVisualizationDependencies {
+  http: any;
+  notifications: any;
```

Comment:
Why do we need these? It does not look like we are using it anywhere.

Reply:
We will be using these two properties in the plugin. So while the props are getting used in the plugin, we ensure they are passed here.

Reply:
in that case dont use `any`. Use this instead:

```
  notifications: CoreStart['notifications'];
  http: CoreStart['http'];
```